tests/lib/path.py

```diff
@@ -20,12 +15,12 @@ class Path(str):
     # Separator in the PATH environment variable.
     pathsep = os.pathsep
 
-    def __new__(cls, *paths):
+    def __new__(cls: Type["Path"], *paths: str) -> "Path":
```

Comment:
You don’t need to type `cls` here; it’s like `self` in `__init__`, Mypy is smart enough to figure it out on its own.